src/System.Management.Automation/utils/FuzzyMatch.cs

```diff
@@ -1,23 +1,38 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Collections.Generic;
 using System.Globalization;
 
 namespace System.Management.Automation
 {
-    internal static class FuzzyMatcher
+    internal class FuzzyMatcher
     {
-        public const int MinimumDistance = 5;
+        internal readonly uint MinimumDistance;
+
+        internal FuzzyMatcher(uint minimumDistance = 5)
```

Comment:
Given that we now expose setting the min distance in the cmdlet, I wonder if we should change this to 3 instead of 5 to get more reasonable results and likely a bucket 3 breaking change

Reply:
It turns out we don't need to declare a default value for this parameter.
The `-FuzzyMinimumDistance` in `Get-Command` is 5 by default, are you suggesting changing it to 3? I think it makes sense, and will make the change.

Reply:
I changed the default min distance back to 5 because the change caused a test to fail -- `pwsh.exe` is expected to return for `pwsw` on Windows by default. I guess we need to improve the fuzzy matcher somehow, to ignore the `.exe` extension 
on the to-be-compared string when the reference string doesn't have `.exe`.